src/bridge/cockpitpackages.c

```diff
@@ -1310,10 +1310,6 @@ cockpit_packages_get_bridges (CockpitPackages *packages)
             {
               g_message ("%s: invalid \"match\" field in package manifest", package->name);
             }
-          else if (match == NULL)
-            {
-              g_message ("%s: missing \"match\" field in package manifest", package->name);
-            }
```

Comment:
Should this at least enforce "match OR privileged", to ensure that a bridge definition still has something that cockpit knows how to activate?

Reply:
Good idea! I'll make this XOR.

Reply:
This means we need to fully remove the pkexec bridge config from the manifest, but I guess that is ok.